toolkit/block-producer-metadata/pallet/src/lib.rs

```diff
@@ -161,12 +165,12 @@ pub mod pallet {
 		type BenchmarkHelper: benchmarking::BenchmarkHelper<Self::BlockProducerMetadata>;
 	}
 
-	/// Storage mapping from block producers to their metadata
+	/// Storage mapping from block producers to their metadata and owner Substrate account
```

Comment:
```suggestion
	/// Storage mapping from block producers to their metadata, owner Substrate account and hold funds.
```
but maybe we can just say about metadata and content describe in the Value type definition